testing/regress/ecl/thor/issue22196.xml

```diff
@@ -0,0 +1,10 @@
+<Exception><Code>100000</Code><Filename>/home/gavin/dev/hpcc/testing/regress/ecl/issue22196.ecl</Filename><Line>4</Line><Source>user</Source><Message>Letter &apos;A&apos; not allowed</Message></Exception>
```

Comment:
test mismatching, because has personal make install path.. 
Can you transform to regex out the leading path, so always matches?

Reply:
We should consider stripping out path information from the filename at compile time - you could argue relative info is valuable but if it's simpler to strip all path info I would not be too upset